src/Scene.js

```diff
@@ -90,37 +77,22 @@ export default class TurtleGraphics extends React.Component<{}, TurtleGraphicsSt
         });
     }
 
-    clear(): void {
-        this.setState({
-            path: []
-        });
-    }
-
     render() {
-        const turtleTransform = `translate(${this.state.location.x} ${this.state.location.y}) rotate(${this.state.directionDegrees} 0 0)`;
+        // Substract 90 degrees from the character bearing as the character image is drawn upright when it is facing East
+        const robotCharacterTransform = `translate(${this.state.location.x} ${this.state.location.y}) rotate(${this.state.directionDegrees - 90} 0 0)`;
 
         return (
             <div>
                 <span
-                    className='TurtleGraphics__drawing-area'
+                    className='Scene__drawing-area'
                     role='img'
                     aria-label='Drawing area'>
```

Comment:
I know that we will be expanding on this in a later task, but for this PR, let's use "Scene" rather than "Drawing area". And let's move the text to our localized messages file.